### PR TITLE
Yask updates

### PR DESCRIPTION
**Pull request by chuckyount**
Use the new YASK v3 APIs where possible.

### CODE REVIEW

File: devito/yask/wrappers.py [hunk 1]
```diff
@@ -301,46 +301,57 @@ def make_grid(self, obj):
...
             elif alloc.put_local:
-                grid.set_numa_preferred(namespace['numa-put-local'])
+                var.set_numa_preferred(namespace['numa-put-local'])
 
         for i, s, h in zip(obj.indices, obj.shape_allocated, obj._size_halo):
             if i.is_Space:
                 # Note:
                 # From the YASK docs: "If the halo is set to a value larger than
                 # the padding size, the padding size will be automatically increased
                 # to accomodate it."
```
Comment (georgebisbas) [inline devito/yask/wrappers.py]: accommodate? :-)

Comment (chuckyount) [reply devito/yask/wrappers.py]: Yep, looks like a typo in the comment. Thanks.

**Comment by FabioLuporini** [comment]
There are [a few pep8 violations](https://travis-ci.org/opesci/devito/jobs/550003079#L3966) , but aside from that, this looks good and I'm happy to land it as is. Thanks!

**Comment by FabioLuporini** [comment]
I was sleepy when I wrote that. What I meant is that once the pep8 violations are gone, we can land this

**Comment by FabioLuporini** [comment]
I might be able to have a go at this once I'm back from holidays

**Comment by chuckyount** [comment]
@FabioLuporini : yes, I was a bit confused by your earlier comment. I'll fix the violations and re-commit.

**Comment by FabioLuporini** [comment]
Merged, thanks

**Pull request merged**